### PR TITLE
For the sbt assembly discarding "cobertura.properties", and testing it.

### PR DESCRIPTION
**Pull request by kshakir**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,5 +6,5 @@ jdk:
 before_script:
   - mysql -e "create database IF NOT EXISTS cromwell_test;"
 
-script: sbt -Dbackend.shared-filesystem.localization.0=copy clean coverage nodocker:test
+script: sbt -Dbackend.shared-filesystem.localization.0=copy clean coverage nodocker:test assembly
```
Comment (scottfrazer) [inline .travis.yml]: Is this so we can see these problems earlier?  Do you think it'll slow down tests a lot?

File: build.sbt [hunk 2]
```diff
@@ -81,7 +81,7 @@ val customMergeStrategy: String => MergeStrategy = {
         MergeStrategy.filterDistinctLines
       case _ => MergeStrategy.deduplicate
     }
-  case "asm-license.txt" | "overview.html" =>
+  case "asm-license.txt" | "overview.html" | "cobertura.properties" =>
```
Comment (scottfrazer) [inline build.sbt]: So this is basically saying "discard all `cobertura.properties` files"?

Comment (mcovarr) [inline build.sbt]: Is there a legit reason the `cobertura.properties` files are included in those jars in the first place?

File: .travis.yml (hunk 1 above)
Comment (kshakir) [inline .travis.yml]: Yes, yes it will slow down tests. "A lot" is relative. I'll leave it to reviewers to decide if we want to leave it in or not.

Comment (scottfrazer) [inline .travis.yml]: I have no preference... it's easy enough to remove if we want to

File: build.sbt (hunk 2 above)
Comment (scottfrazer) [inline build.sbt]: Is it a default setting to include JAR files from dependencies?

Comment (kshakir) [inline build.sbt]: Yup.

Here are the contents of the two current `cobertura.properties` files we're discarding and not merging, etc.:

```bash
unzip -p \
  ~/.ivy2/cache/com.google.gcloud/gcloud-java-core/jars/gcloud-java-core-0.0.9.jar \
  cobertura.properties
```
```
#Generated by maven-cobertura-plugin for project com.google.gcloud:gcloud-java-core:jar:0.0.9
#Fri Oct 09 00:08:23 UTC 2015
net.sourceforge.cobertura.datafile=/home/travis/build/GoogleCloudPlatform/gcloud-java/gcloud-java-core/target/cobertura/cobertura.ser
```

versus

```
unzip -p \
  ~/.ivy2/cache/com.google.gcloud/gcloud-java/jars/gcloud-java-0.0.9.jar \
  cobertura.properties
```
```
#Generated by maven-cobertura-plugin for project com.google.gcloud:gcloud-java:jar:0.0.9
#Fri Oct 09 00:09:20 UTC 2015
net.sourceforge.cobertura.datafile=/home/travis/build/GoogleCloudPlatform/gcloud-java/gcloud-java/target/cobertura/cobertura.ser
```

Comment (kshakir) [inline build.sbt]: > Is there a legit reason the cobertura.properties files are included in those jars in the first place?

Not in my humble opinion.

> Is it a default setting to include JAR files from dependencies?

Yes, all files are "assembled" into the executable jar.

**Comment by kshakir** [comment]
@scottfrazer First reviewer

**Comment by scottfrazer** [comment]
:+1: @mcovarr second reviewer

**Comment by mcovarr** [comment]
:+1:

**Pull request merged**